docs/getting-start.md

```diff
@@ -79,203 +79,207 @@ required for testing locally but is required for testing on your personal fork.
 1. Navigate to CloudFormation from aws console
 2. Choose file upload.
 3. Upload the following as a file
-```
-Parameters:
-  GitHubOrg:
-    Type: String
-  RepositoryName:
-    Type: String
-  OIDCProviderArn:
-    Description: Arn for the GitHub OIDC Provider.
-    Default: ""
-    Type: String
-
-Conditions:
-  CreateOIDCProvider: !Equals 
-    - !Ref OIDCProviderArn
-    - ""
-
-Resources:
-  Role:
-    Type: AWS::IAM::Role
-    Properties:
-      AssumeRolePolicyDocument:
-        Statement:
-          - Effect: Allow
-            Action: sts:AssumeRoleWithWebIdentity
-            Principal:
-              Federated: !If 
-                - CreateOIDCProvider
-                - !Ref GithubOidc
-                - !Ref OIDCProviderArn
-            Condition:
-              StringLike:
-                token.actions.githubusercontent.com:sub: !Sub repo:${GitHubOrg}/${RepositoryName}:*
-
-  GithubOidc:
-    Type: AWS::IAM::OIDCProvider
-    Condition: CreateOIDCProvider
-    Properties:
-      Url: https://token.actions.githubusercontent.com
-      ClientIdList: 
-        - sts.amazonaws.com
-      ThumbprintList:
-        - 6938fd4d98bab03faadb97b34396831e3780aea1
-
-Outputs:
-  Role:
-    Value: !GetAtt Role.Arn 
-```
+    ```
+    Parameters:
+      GitHubOrg:
+        Type: String
+      RepositoryName:
+        Type: String
+      OIDCProviderArn:
+        Description: Arn for the GitHub OIDC Provider.
+        Default: ""
+        Type: String
+      AccountRoot:
+        Description: "This account's ID to allow assume, in the form arn:aws:iam:[account ID]:root"
+        Type: String
+    
+    Conditions:
+      CreateOIDCProvider: !Equals 
+        - !Ref OIDCProviderArn
+        - ""
+    
+    Resources:
+      Role:
+        Type: AWS::IAM::Role
+        Properties:
+          AssumeRolePolicyDocument:
+            Statement:
+              - Effect: Allow
+                Action: sts:AssumeRoleWithWebIdentity
+                Principal:
+                  Federated: !If 
+                    - CreateOIDCProvider
+                    - !Ref GithubOidc
+                    - !Ref OIDCProviderArn
+                Condition:
+                  StringLike:
+                    token.actions.githubusercontent.com:sub: !Sub repo:${GitHubOrg}/${RepositoryName}:*
+              - Sid: AssumeFromRoot
```

Comment:
I was just fixing the formatting here in the markdown, and added the additional setup needed to assume role. Because the steps currently don't give access to assume the terraform role that creates the cluster, so there's no access to the eks cluster